docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -0,0 +1,65 @@
+# Using devices with Clear Containers
+
+Host devices can be passed to a container with Docker using `--device` option
```

Comment:
Lines 3-4 rewrite: "You can use the docker run command with the `--device` option to pass host devices to a container. The following example passes a device:"